### PR TITLE
Active record feature

### PR DESCRIPTION
**Pull request by AleksandarIvezic**
### In this project we : 
- Created simplified Reddit 
- Add models for Users, Posts, and Comments
- Add associations 
- Add validations
- Used Ruby on Rails on this project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,40 @@
-# README
+# Micro Reddit
+In this project we practice our knowledge of Active records, models, associations and validatios with Ruby on Rails. It is a simplified version of Reddit which allows users to make posts and add comments to these posts. The description of this project can be found [here](https://www.theodinproject.com/paths/full-stack-ruby-on-rails/courses/ruby-on-rails/lessons/building-with-active-record-ruby-on-rails).
```
Comment (rindrajosia) [inline README.md]: Here for example, change **validatios** to **validation**

File: README.md [hunk 2]
```diff
@@ -1,24 +1,40 @@
...
+Clone this repository to get a copy of this project. 
+- Open your terminal
+- Run `git clone https://github.com/blakbox23/micro-reddit.git`
+- Open the project with your favourite code editor.
 
-* Ruby version
+## Installation
+After cloning the repo, you also need to install required gems in your machine.
+- Run `bundle install` to install the dependancies.
+- Run `rails db:migrate` to create required database tables.
```
Comment (rindrajosia) [inline README.md]: Before migration, you should ask the user to create DB via **rails db:create.**

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Changes Required :red_circle:
To Highlight: :angel:

- A user can be created :+1:
- A post can be created :100:
- A comment can be created :1st_place_medal:

To improve :factory_worker: [REQUIRED]:

- [x]  Please check your readme by using tools like grammarly because it is the first thing user checks. (check inline comments below :arrow_double_down:)

Nice to have [OPTIONAL] :

- [ ] 

That's it, you can submit another code review request after making these changes :recycle: :innocent:
With all my respect :pray:
Rindra Josia

**Review by mke2111 — APPROVED** [review]
# 😄 🥇  STATUS: APPROVED  🥇 👏🏽 

## Hey Happy Coders, 👋🏽 
Congratulations, 🥂 . You got minimum requirements for approval. 👍🏽 

### **_Well Done_** 💯 

    1. The readme is now descriptive enough and grammatically correct
    2. The linter checks pass successfully.

### You may now merge to the develop branch and remember to submit a project completion form.
### I wish you the very best in your upcoming projects.

**Cheers 🍹 and Happy coding 💻** 

<hr>

### Best regards
Slack 🆘  Roy Mukuye
Twitter 🐦  [Roy Mukuye](https://twitter.com/Roymkenya)
LinkedIn 🔗  [Roy Mukuye](https://www.linkedin.com/in/roy-mukuye/)

**Pull request merged**